smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -45,17 +45,33 @@ def setUp(self):
             for tmp in Section.interfaces("ethernet"):
                 if not '.' in tmp:
                     self._members.append(tmp)
+            
+            self.session.set(['interfaces','dummy','dum0'])
+            self.session.set(['interfaces','dummy','dum1'])
+            self.session.commit()
```

Comment:
Why is this `commit()` required?

Reply:
Mainly to make the required interface effective during testing